ui/src/pages/sitemaps/other.xml.tsx

```diff
@@ -0,0 +1,130 @@
+import fs from 'fs';
+import client from '@contentfulClient';
+
+import * as Interfaces from '@interfaces';
+import * as Types from '@types';
+import * as api from '@api';
+import * as Config from '@config';
+
+const Sitemap = () => {
+    return null;
+};
+
+// Return a sitemap with URLs of all static pages, topics, and blog posts.
+export const getServerSideProps: Types.GetServerSideProps = async ({ res }) => {
+    const now = new Date().toISOString();
+    // Get URLs of all static pages by scanning file system.
+    const staticUrls = fs
+        .readdirSync(process.cwd() + '/src/pages')
+        .filter(
+            (staticPage) =>
+                ![
+                    '404.tsx',
+                    '500.tsx',
+                    '_app.tsx',
+                    '_document.tsx',
+                    'account.tsx',
+                    'api',
+                    'author-link.tsx',
+                    'authors',
+                    'login.tsx',
+                    'my-bookmarks.tsx',
+                    'publications',
+                    'publications-sitemap-index.xml.tsx',
+                    'publications-sitemap[number].xml.tsx',
+                    'sitemap.xml.tsx',
+                    'topics',
+                    'verify.tsx'
+                ].includes(staticPage)
```

Comment:
We may need to refine this filter a bit to also catch paths starting with period ".", "approve-control-request.tsx"
![image](https://github.com/JiscSD/octopus/assets/48569671/12914c67-5fb3-4718-a725-a6867d0c0cf5)

Reply:
I've adjusted this list so that it will exclude files starting with '.' and 'approve-control-request.tsx'